src/basic/time-util.c

```diff
@@ -55,6 +56,17 @@ usec_t now(clockid_t clock_id) {
         return timespec_load(&ts);
 }
 
+usec_t epoch_or_now(clockid_t clock_id) {
+        int r;
+        unsigned long long epoch;
+
+        r = getenv_llu_secure("SOURCE_DATE_EPOCH", 10, &epoch);
+        if (r == 0)
```

Comment:
Please use `>= 0` rather than `== 0`, and not necessary to assign a value to `r`:
```c
if (getenv_llu_secure(...) >= 0)
        return (usec_t) epoch * USEC_PER_SEC;
```

Reply:
Thanks, done.